swmm_views/vw_swmm_conduits.sql

```diff
@@ -0,0 +1,35 @@
+DROP VIEW IF EXISTS qgep_swmm.vw_conduits;
```

Comment:
This sounds French, is this the SWMM standard or something particular to the qgis swmm plugin?

Reply:
conduits is the name used in SWMM